packages/core/utils/src/dml/__tests__/entity-builder.spec.ts

```diff
@@ -180,6 +207,26 @@ describe("Entity builder", () => {
           getter: false,
           setter: false,
         },
+        spend_limit: {
+          columnType: "numeric",
+          default: 500.4,
+          getter: true,
+          name: "spend_limit",
+          nullable: false,
+          reference: "scalar",
+          setter: true,
+          trackChanges: false,
+          type: "any",
```

Comment:
@adrien2p what do you think the type should be here for this and raw_ field?

Reply:
I think we can keep it as any cause I am not sure how to tell mikro orm it can be BigNumberInput. what they do it that they have a util which takes the type of the input as follow and compare it. so any seems the best for now
```ts
const objectType = Object.prototype.toString.call(value);
return objectType.match(/\[object (\w+)]/)[1].toLowerCase();
```